appcast/releasenotes1010.html

```diff
@@ -0,0 +1,49 @@
+<html>
+
+	<head>
+		<meta http-equiv="content-type" content="text/html;charset=utf-8">
+		<title>What's new in Transmission?</title>
+		<meta name="robots" content="anchors">
+		<link href="releasenotes.css" type="text/css" rel="stylesheet" media="all">
+	</head>
+
+	<body>
+			<table class="dots" width="100%" border="0" cellspacing="0" cellpadding="0" summary="Two column table with heading">
+				<tr>
+					<td class="red">
+						<h3>Note</h3>
+					</td>
+				</tr>
+				<tr>
+					<td>
+						<p>If auto-updating fails, you can download directly from <a href="https://transmissionbt.com">transmissionbt.com</a>. Sorry about that!</p>
+					</td>
+				</tr>
+			</table>
+			
+			<br>
+
+			<table class="dots" width="100%" border="0" cellspacing="0" cellpadding="0" summary="Two column table with heading">
+				<tr>
+					<td class="blue">
+						<h3>3.00 App Changes</h3>
+					</td>
+				</tr>
+				<tr>
+					<td>
+						<ul>
+						<li>Dark Mode support</li>
+						<li>Fix autoupdate on High Sierra and up by bumping the Sparkle version</li>
+						<li>Allow the RPC server to listen on an IPv6 address</li>
+						<li>Use proper UTF-8 encoding (with macOS-specific normalization) when setting download/incomplete directory and completion script path</li>
+						<li>Remove Growl support, notification center is always used</li>
+						</ul>
+					</td>
+				</tr>
+			</table>
+			
+			<p>Transmission requires macOS 10.10 or better.</p>
```

Comment:
```suggestion
			<p>Transmission requires macOS 10.13 or better.</p>
```

Reply:
This is the release notes file for 3.0. I just copied it over before updating the original for 4.0.